pkg/codegen/dotnet/gen.go

```diff
@@ -663,12 +711,8 @@ func (mod *modContext) genResource(w io.Writer, r *schema.Resource) error {
 	switch len(r.Aliases) {
 	case 0:
 		fmt.Fprintf(w, "\n")
-	case 1:
-		fmt.Fprintf(w, "                Aliases = { ")
-		genAlias(w, r.Aliases[0])
-		fmt.Fprintf(w, " },\n")
 	default:
-		fmt.Fprintf(w, "                Aliases =\n")
+		fmt.Fprintf(w, "\n                Aliases =\n")
```

Comment:
This is the only change that affects TF providers. I prefer this alignment over existing one.